elf/main.cc

```diff
@@ -373,7 +373,7 @@ static int redo_main(int argc, char **argv, std::string_view target) {
 template <typename E>
 int elf_main(int argc, char **argv) {
   Context<E> ctx;
-
+  using namespace std::string_view_literals;
```

Comment:
I don't think this is necessary because we have it in `mold.h` in the top directory